lib/createHtml.jsx

```diff
@@ -61,6 +69,11 @@ export default function(options) {
   const googleTagManagerId = options && options.googleTagManagerId || null;
   const visualWebsiteOptimizer = options && options.visualWebsiteOptimizer || false;
 
+  let vwoAccountId = 215379;
+  if (visualWebsiteOptimizer && typeof(visualWebsiteOptimizer) === "object" && visualWebsiteOptimizer.accountId) {
```

Comment:
don't need both `visualWebsiteOptimizer && typeof(visualWebsiteOptimizer) === "object"`, the second covers the first case.